fuse_core/include/fuse_core/parameter.h

```diff
@@ -66,7 +66,7 @@ void getParamRequired(const ros::NodeHandle& nh, const std::string& key, T& valu
 }
 
 /**
- * @brief Helper function that loads strictly positive integral or floating point values from the parameter server
```

Comment:
I forgot to update the doxygen in the previous PR.

I was trying to support `getPositiveParam` for `ros::Duration`, but it requires too much boilerplate, so I've given up. :smile: